utils/rpms/daos.spec

```diff
@@ -33,6 +33,7 @@ BuildRequires: scons >= 2.4
 %endif
 BuildRequires: libfabric-devel >= %{libfabric_version}
 BuildRequires: mercury-devel >= %{mercury_version}, mercury-devel < %{mercury_max_version}
+BuildConflicts: mercury-ucx
```

Comment:
This isn't necessary.  There is no linkage between `mercury-ucx` and `mercury-devel`.

Reply:
ok I removed it